initial_website_build/initial_website_build.info.yml

```diff
@@ -0,0 +1,29 @@
+name: 'Initial website build'
```

Comment:
I'd probably change this to `Website Configuration` and rename the package to `website_configuration`